templates/federated_reporting/import_file.sh

```diff
@@ -28,15 +28,12 @@ mv "$file" "$file.importing"
   cat<<EOF
 \set ON_ERROR_STOP 1
 BEGIN;
-SELECT ensure_feeder_schema('$hostkey', ARRAY[$table_whitelist]);
```

Comment:
the call to ensure_feeder_schema() could stay here without any problem. There is nothing there that creates a lock other than the momentary ALTER SCHEMA RENAME TO which should happen quite fast.

Reply:
> which should happen quite fast

This is exactly how deadlocks are usually born. I suggest that we do all the schema changes serially. As you say, they should be quite fast so there's no reason to do them in parallel risking issues. If we later find out that we need to optimize this, parallelization may be one of the options.